python_src/tests/performance_manager/test_performance_manager.py

```diff
@@ -174,64 +174,160 @@ def mock__get_static_parquet_paths(
         mock__get_static_parquet_paths,
     )
 
-    # pylint: disable=R0913
-    # pylint too many arguments (more than 5)
-    def mock__write_parquet_file(
-        table: Table,
-        file_type: str,
-        s3_dir: str,
-        partition_cols: List[str],
-        visitor_func: Optional[Callable[..., None]] = None,
-        basename_template: Optional[str] = None,
-    ) -> None:
+    yield
+
+
+@pytest.fixture(name="_flat_file_s3_patch")
```

Comment:
i created a fixture just for the s3 flat file interactions.

this one will only be used on tests using that functionality.